docs/developer/soffits/README.md

```diff
@@ -0,0 +1,46 @@
+# uPortal Soffits
+
+Soffit is a technology for creating content that runs in [Apereo uPortal][].  It
+is intended as an alternative to [JSR-286 portlet development][].
+
+## Why Would I Want Soffits?
+
+You are a Java web application developer.  You are tasked with developing content
+for [Apereo uPortal][].
+
+You are not excited about doing Java Portlet development
+[in the traditional way][] or even using [Spring Portlet MVC][].  You correctly
+conclude that the Java Portlet APIs are _large_, _obtuse_, and _actively
+interfere_ with contemporary web development practices and frameworks that you
+want to be using.
+
+Apereo Soffit is an alternative approach to producing content for uPortal that
+is not based on JSR-286 or the portlet container.
```

Comment:
Soffits convey mode and window state, so are conceptually based on `JSR-286` and the idioms of a portlet container.